website/docs/docs/partnerships.mdx

```diff
@@ -148,23 +148,21 @@ Work with your HashiCorp Alliances representative to verify the plugin within th
 
 ### 5. Support
 
-Getting a new provider built and published to the Terraform Registry is just the first step towards enabling your users with a quality Terraform integration. Once a verified provider has been published, on-going effort is required to maintain the provider.
+Getting a new provider built and published to the Terraform Registry is just the first step towards enabling your users with a quality Terraform integration. Once a partner's provider has been published, on-going effort is required to maintain the provider.
 
-HashiCorp Terraform has an extremely wide community of users and contributors and we encourage everyone to report issues however small, as well as help resolve them when possible. We expect that all verified provider publishers will continue to maintain the provider and address any issues users report in a timely manner. This includes resolving all critical issues within 48 hours and all other issues within 5 business days. HashiCorp reserves the right to remove verified status from any integration that is no longer being maintained.
+HashiCorp Terraform has an extremely wide community of users and contributors and we encourage everyone to report issues however small, as well as help resolve them when possible. We expect that all partner provider publishers will continue to maintain the provider and address any issues users report in a timely manner. This includes resolving all critical issues within 48 hours and all other issues within 5 business days. HashiCorp reserves the right to remove partner status from any integration that is no longer being maintained.
```

Comment:
I was thinking that "is not being maintained" is kind of vague. What if we said "that doesn't meet our maintenance requirements" --> Thought is someone could maybe argue they ARE maintaining it....but if it's not up to our expectations, we still can pull it. Thoughts?

```suggestion
HashiCorp Terraform has an extremely wide community of users and contributors, and we encourage the community to report issues and help resolve them. However, we expect that all partner provider publishers continue maintaining the provider and that they address user-reported issues in a timely manner. This includes resolving all critical issues within 48 hours and all other issues within 5 business days. HashiCorp reserves the right to remove partner status from any integration that does not meet our maintenance requirements.
```

Reply:
I think so, makes sense.